### ISSUE TITLE
typos/ inconsistent naming

### ISSUE THREAD

**Issue by EdJoPaTo** [Transport-Protocol/CaDSPracticalExamVS#11]
``` java
IIDLCaDSEV3RMIMoveHorizontally
IIDLCaDSEV3RMIMoveVertical
```
- should be adverb (horizontally) or adjective (horizontal) and not both

---

``` java
ICADSRMIComsumer
```
- coMsumer should be coNsumer
- cAds A should be lower case like everywhere else

---

``` java
ICaDsEV3StatusListener
```
- cadS S should be upper case like everywhere else

---

``` java
ICaDsEV3StatusListener
ICaDSGripperFeedBack
```

both are Interfaces to handle status/ feedback from the gripper
- both Listener or both not Listener suffix
- both EV3 or both Gripper in prefix

---

``` java
caller.moveRight();
caller.moveLeft();
caller.moveUP();
caller.moveDown();
```
- `moveUP()` should be written with a lowercase p -> `moveUp()`

**Comment by scimbe** [comment]
Typos fixed in 74eee11db8250208a8f56825db8686aacdfb6ed7

**Comment by scimbe** [comment]
merci
